playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -56,3 +56,12 @@
   - import_role:
       name: openshift_web_console
       tasks_from: remove_old_asset_config
+
+- hosts: oo_first_master
```

Comment:
@michaelgugino I did like that because migrate_default_registry_var.yml  was done so.
Can you show an example or documentation of how to do it correctly?

https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/upgrades/v3_9/upgrade_control_plane.yml#L149-L156

Reply:
I wasn't aware we did that there.

Since this is a one-time fix, I suppose it makes sense to keep in in the v3_10 area and this file is fine.  Let's add a comment to remove for 3_11 so we don't forget.

Reply:
i would not like to consider this a one time fix.  We may need additional migrations in the future, i'd like to define a proper pattern for them while we are introducing this.

and i wouldn't intend to remove this for 3.11 (though strictly speaking it should not be needed once a customer has gone through a 3.10 upgrade)

Reply:
@bparees @michaelgugino Where I should put this additional migration playbooks ? I know we have a playbook for [redeploy certificates](https://github.com/openshift/openshift-ansible/blob/master/playbooks/redeploy-certificates.yml), but this playbook is for regular use. The playbook that you suggect is not like that. It will be called sometimes.

Reply:
> ince this is a one-time fix, I suppose it makes sense to keep in in the v3_10 area and this file is fine. Let's add a comment to remove for 3_11 so we don't forget.

@michaelgugino I added a comment. So it looks better for you?